app/pages/about/highlights.jsx

```diff
@@ -9,30 +9,41 @@ class Highlights extends React.Component {
         <Markdown>{counterpart('about.highlights.title')}</Markdown>
         <Markdown>{counterpart('about.highlights.paragraphOne')}</Markdown>
         <Markdown>{counterpart('about.highlights.paragraphTwo')}</Markdown>
+        <Markdown>{counterpart('about.highlights.paragraphThree')}</Markdown>
+        <Markdown>{counterpart('about.highlights.sectionHeader', { volumeNumber: '3', year: '2021' })}</Markdown>
+        <div className="highlights__book-section">
+          <img
+            alt={counterpart('about.highlights.imageAlt', { year: '2021' })}
+            className="highlights__book-image"
+            src="/assets/highlights-book-2021.png"
+          />
+          <div className="highlights__book-description">
+            <Markdown>{counterpart('about.highlights.toDownload', { url: 'https://bit.ly/zoonibook21-download-pdf' })}</Markdown>
+            <Markdown>{counterpart('about.highlights.toPurchase', { purchaseSource: 'Blurb.com', url: 'https://bit.ly/zoonibook21-buy' })}</Markdown>
```

Comment:
Consider having this url open in a new tab. It is my opinion that External sites should open in a new tab.

Reply:
Not entirely sure, but I think our general policy is to open links in the current tab, similar to links in https://www.zooniverse.org/about/publications and https://www.zooniverse.org/about/resources. I think it's based on [the W3C guidelines](https://www.w3.org/TR/WCAG20-TECHS/G200.html) which [this blog post](https://codersblock.com/blog/external-links-new-tabs-and-accessibility/#accessibility-implications) explains a bit more.

Reply:
If you open in a new tab, either add `(opens a new tab)` to the end of the link text or use our external link icon, with `opens in a new tab` as the alt text. Otherwise, it could be confusing for blind readers, or for people who expect the browser's Back button to take them back to Zooniverse.